src/main/java/com/blockout/PaneParams.java

```diff
@@ -1,5 +1,6 @@
 package com.blockout;
 
+import com.minecolonies.util.Log;
```

Comment:
I'll  "copy" the logger of minecolonies to blockout and will use "blockout" as mod id.
After midday meal =D